src/components/ChartCell.js

```diff
@@ -0,0 +1,92 @@
+// @flow
+'use strict';
+
+import React, { PropTypes } from 'react';
+import {
+  StyleSheet,
+  Text,
+  TouchableHighlight,
+  View
+} from 'react-native';
+import Icon from 'react-native-vector-icons/Ionicons';
+import Colors from '../styles/Colors';
+import FontStyles from '../styles/FontStyles';
+
+const displayText = false;
+
+const ChartCell = (props) => {
+  var favIcon = 'ios-heart-outline';
+  if(props.vfrChart.isFavorited == true){
+    favIcon = 'ios-heart';
+  }
+      return (
+        <View style={styles.inputsContainer}>
+            <View style={styles.vfrText}>
+              <TouchableHighlight style={styles.textHighlight} onPress={() => console.log('eventually display more map information')}>
```

Comment:
color change looks nice! and the idea was when they pressed the arrow it would take them to the view (now it just goes to settings) and when they pressed on any part of the view it would toggle more information about the chart thats the next thing i was gonna implement just wanted to put this in first